RELEASING

```diff
@@ -0,0 +1,25 @@
+# Scala.js Release Process
+
+1. Clean up bugs:
+    - Make sure all bugs assigned to the milestone are closed.
+    - Make sure all fixed bugs are assigned to the milestone ([this query][1]
+      should not return anything).
```

Comment:
> Can't we just use "invalid" for separate repo?

I don't think that's accurate. #2168 and #2039 were perfectly valid requests, which we actually fixed, by providing them in a separate repo. There was therefore no milestone associated with those issues, but they were definitely not valid, and closed as fixed.